File: Sort/Bubble_Sort.py
#defining the function bubblesort
def bubblesort(list):

# Swap the elements to arrange in order
    for iter_num in range(len(list)-1,0,-1):
        
        # Traverse through all array elements
        for index in range(iter_num):

            # Swap if the element found is greater 
            # than the next element using a temporary variable.
            if list[index]>list[index+1]:
                temp = list[index]
                list[index] = list[index+1]
                list[index+1] = temp

#the given unsorted list(input list)
list = [19,2,31,45,6,11,121,27]
print(f"The given unsorted list: {list}")

#the output sorted list.
bubblesort(list)
print(f"The sorted list is: {list}")

File: Sort/bubble_sort.java


import java.util.*;
import java.lang.*;
import java.io.*;


class bubble_sort
{
    public static void main (String[] args) throws java.lang.Exception
    {
        Scanner sc=new Scanner(System.in);
        int n=sc.nextInt();
        int a[]=new int[n];
        int i,j,temp;
        
        for(i=0;i<n;i++)
        {
            a[i]=sc.nextInt();
        }
        for(i=0;i<n-1;i++)
        {
            for(j=0;j<n-i-1;j++)
            {
                if(a[j]>a[j+1]){
                    temp=a[j];
                    a[j]=a[j+1];
                    a[j+1]=temp;
                }
            }
        }
        System.out.println("Sorted array:");
        for(i=0;i<n;i++)
        {
            System.out.print(a[i]+" ");
        }
    }
}
